tests/upgrade/postgres_run.sh

```diff
@@ -105,6 +105,13 @@ test_upgrade_paths() {
 
     cd "$TEST_ROOT"
 
+    # This test does a lot of upgrades and bounces.  If things take a little longer to bounce we can get entries in
```

Comment:
I like it! But this will result in a `-dirty` tag. Which may not be a problem if the tag is already consumed but could cause future headache. What about allowing `ALLOWLIST_FILE` to be already set in `scripts/ci/logcheck/check.sh`?  And then set it up for that call i.e. `cp scripts/ci/logcheck/allowlist-patterns /tmp/allowlist-patterns; echo ... >> /tmp/allowlist-patterns; export ALLOWLIST_FILE="/tmp/allowlist-patterns"`